src/execution/filter_executor.cpp

```diff
@@ -0,0 +1,13 @@
+#include "execution/executors/filter_executor.h"
+
+namespace bustub {
+
+FilterExecutor::FilterExecutor(ExecutorContext *exec_ctx, const FilterPlanNode *plan,
+                               std::unique_ptr<AbstractExecutor> &&child_executor)
+    : AbstractExecutor(exec_ctx) {}
+
+void FilterExecutor::Init() {}
```

Comment:
Thoughts on making skeletons raise exceptions saying they're not implemented in Init?